### PR TITLE
Feature/production certificates

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,32 +1,51 @@
-@Library('global-jenkins-library@2.2.3') _
+@Library('global-jenkins-library@feature/docker-build-options') _
```
Comment (jeremyjams) [inline Jenkinsfile:1]: To update before merge

File: Jenkinsfile [hunk 2]
```diff
@@ -1,32 +1,51 @@
...
-        buildInfo: buildInfo,
-        integrationTestsEnvVars: [],
-        shouldPublishJars: false,
-        shouldPublishDockerImages: true,
-        dockerfileDir: '.',
-        buildContext: '.',
-        dockerImageRepositoryName: repositoryName,
-        preProductionVisibility: 'docker.io',
-        productionVisibility: 'docker.io')
+// buildJavaProject(
```
Comment (jeremyjams) [inline Jenkinsfile:7]: All to enable before merge

File: gramine/Dockerfile [hunk 3]
```diff
@@ -17,7 +17,7 @@ RUN ls -al /home/build/libs
 ### Build the final graminized image with
 ### the produced jar.
 ###
-FROM docker-regis.iex.ec/iexec-graphene-base:0.7.0
+FROM docker-regis.iex.ec/iexec-graphene-base:feature-bad30168
```
Comment (jeremyjams) [inline gramine/Dockerfile:20]: To update before merge

File: Jenkinsfile [hunk 4]
```diff
@@ -1,32 +1,51 @@
...
 
-sconeBuildUnlocked(
-        nativeImage:     "docker-regis.iex.ec/$repositoryName:$buildInfo.imageTag",
-        imageName:       repositoryName,
-        imageTag:        buildInfo.imageTag,
-        sconifyArgsPath: './docker/sconify.args',
-        sconifyVersion:  '5.7.1'
-)
+// sconeBuildUnlocked(
+//         nativeImage:     "docker-regis.iex.ec/$repositoryName:$buildInfo.imageTag",
```
Comment (jbern0rd) [inline Jenkinsfile:46]: Beware, `buildInfo.imageTag` will have changed here ! (will have `-test-ca` suffix)
Add `buildInfo.imageTag -= '-test-ca'`

File: Jenkinsfile [hunk 5]
```diff
@@ -1,32 +1,51 @@
...
+        productionImageName = buildSimpleDocker_v3(
+            buildInfo: buildInfo,
+            dockerfileDir: dockerfileDir,
+            buildContext: '.',
+            dockerImageRepositoryName: dockerImageRepositoryName,
+            visibility: visibility
+        )
+    }
+    stage('Build Gramine test CA Gramine image') {
+        buildInfo.imageTag = buildInfo.imageTag + '-test-ca'
```
Comment (jbern0rd) [inline Jenkinsfile:33]:
```suggestion
        buildInfo.imageTag += '-test-ca'
```

**Review by jbern0rd — APPROVED** [review]

**Review by mcornaton — APPROVED** [review]

**Pull request merged**